en/docs/chapter_divide_and_conquer/binary_search_recur.md

```diff
@@ -1,40 +1,40 @@
 
-In fact, **search algorithms with a time complexity of $O(\log n)$ are usually based on the divide-and-conquer strategy**, such as binary search and trees.
+In fact, **search algorithms with a time complexity of $O(\log n)$ are typically based on the divide-and-conquer strategy**, such as binary search and tree.
 
-- Each step of binary search divides the problem (searching for a target element in an array) into a smaller problem (searching for the target element in half of the array), continuing until the array is empty or the target element is found.
-- Trees represent the divide-and-conquer idea, where in data structures like binary search trees, AVL trees, and heaps, the time complexity of various operations is $O(\log n)$.
+- Binary search divides the original problem (searching for a target element in an array) into a smaller subproblem (searching for a target element in half of the array) at each step. This process continues until the array is empty or the target element is found.
+- Trees are a classic example of the divide-and-conquer paradigm. In data structures such as binary search trees, AVL trees, and heaps, the time complexity of operations is $O(\log n)$.
 
-The divide-and-conquer strategy of binary search is as follows.
+The divide-and-conquer strategy of binary search is characterized as follows.
 
-- **The problem can be divided**: Binary search recursively divides the original problem (searching in an array) into subproblems (searching in half of the array), achieved by comparing the middle element with the target element.
-- **Subproblems are independent**: In binary search, each round handles one subproblem, unaffected by other subproblems.
-- **The solutions of subproblems do not need to be merged**: Binary search aims to find a specific element, so there is no need to merge the solutions of subproblems. When a subproblem is solved, the original problem is also solved.
+- **Dividing the problem**: Binary search recursively divides the original problem (searching in the full array) into subproblems (searching in half of the array) by comparing the middle element with the target element.
+- **Independence of subproblems**: Each round of binary search handle only one subproblem, unaffected by other subproblems.
+- **No subproblem solution merging**: Binary search aims to find a specific element, so there is no need to merge the solutions of subproblems. Solving the subproblem inherently solves the original problem.
 
-Divide-and-conquer can enhance search efficiency because brute-force search can only eliminate one option per round, **whereas divide-and-conquer can eliminate half of the options**.
+Divide-and-conquer improves search efficiency. This is because, in brute-force search, only one option can be eliminated per round. In contrast, **divide-and-conquer search eliminates half of the options in each round**.
```

Comment:
"This is because, in brute-force search" -> "Since in brute-force search" might be more concise.